ql/src/test/queries/clientpositive/join_using_clause.q

```diff
@@ -0,0 +1,69 @@
+
+create table test (
+ a int
+);
+
+insert into test values (1);
+
+create table test1 (
+ a int,
+ b int
+);
+
+insert into test1 values (1, 2);
+
+create table test2 (
+ a int,
+ b int,
+ c int
+);
+
+insert into test2 values (1, 2, 3);
+
+-- self join with 1 column
+select * from test t1
+join test t2 using(a)
+join test t3 using(a);
+
+explain extended select * from test t1
```

Comment:
`EXPLAIN  EXTENDED` is quite verbose. Do we really need it to prove that the NPE is fixed? If we need to verify that plan is correct isn't `EXPLAIN CBO` sufficient?

Reply:
I agree that it is very verbose! But I wanted to get the OPTIMIZED SQL that shows the on clause transformation. Is there another way to print the OPTIMIZED SQL?

Reply:
The OPTIMIZED SQL is generated from the CBO plan so displaying the latter is sufficient. If in the CBO plan you have `Join(condition=[something])` then you will see that something generated as a `ON` clause when you go from plan to OPTIMIZED SQL (i.e., `RelToSqlConverter`).